tools/migrations/24-12-13--adding-user-commitment-table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE user_commitment (
+    user_id INT PRIMARY KEY,
+    user_minutes INT,
```

Comment:
there is no reason to prefix the `minutes`, `days` with `user`. we already have the info that this table pertains to a user in the table name.